example/lib/pages/tile_loading_error_handle.dart

```diff
@@ -50,7 +50,7 @@ class _TileLoadingErrorHandleState extends State<TileLoadingErrorHandle> {
                       tileProvider: const NonCachingNetworkTileProvider(),
                       errorTileCallback: (Tile tile, error) {
                         if (_needLoadingError) {
-                          WidgetsBinding.instance.addPostFrameCallback((_) {
+                          WidgetsBinding.instance?.addPostFrameCallback((_) {
```

Comment:
on master branch of flutter `WidgetsBinding.instance` is now not nullable (that's why this error occured)